src/data.js

```diff
@@ -46,7 +46,12 @@ export const biography = {
     id: "Predator Type",
     value: "Alleycat",
     list: ["Alleycat", "Bagger", "Blood Leech", "Cleaver", "Consensualist", "Farmer", "Osiris", "Sandman", "Scene Queen", "Siren"],
-    disabledValue: "Choose Your Predator Type"
+    disabledValue: "Choose Your Predator Type",
+  },
+  predatorDiscipline: {
+    id: "Predator Discipline",
+    list: ["Celerity", "Potence"],
```

Comment:
tuto sa mi stale nezda ten list, a to ze ho musis menit apod, preco to je vlastne samostatny objekt a neni pod predator type?

Reply:
lebo predatorType ma svoju value and predatorDiscipline ma tiez svoju value? 

Preco by to malo byt naopak?